### PR TITLE
Feature/58 question translation and tooltip

### LINKED ISSUE #58: Question Translation and Tooltip
## TODO/DONE

Translation for
- [ ] Question Text
- [ ] Tooltip text
- [ ] Options
- [ ] Error message

### Test cases

- [ ]  should translate Question Text
- [ ] should translate tooltip text
- [ ]  should translate Options
- [ ]  should translate Error message with Yup
- [ ]  should update translation on language change
- [ ]  should fallback to default language when translation is not available

### PR DESCRIPTION
**Pull request by ifirmawan**
## TODO/DONE

Translation for
- [x] Question Text
- [x] Tooltip Text
- [x] Options

### Test cases

- [x]  should translate Question Text
- [x] should translate tooltip text
- [x]  should translate Options
- [x]  should update translation on language change
- [x]  should fallback to default language when translation is not available

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204868663470086

### CODE REVIEW

**Comment by wayangalihpratama** [comment]
Hi mas @ifirmawan, 
I saw we do the translation inside each of the field types.
How if we do something like we implement on akvo react form, something like this one
https://github.com/akvo/akvo-react-form/blob/d68606f2e6c6534b6900edb3f71c41a7b33afbaa/src/lib/index.js#L23-L79

I think we can handle the translation in one place, instead of doing that inside each field types.

**Review by wayangalihpratama — COMMENTED** [review]
I have a comment in general. 
How if we implement the translations like we do on akvo react form 
https://github.com/akvo/akvo-react-form/blob/d68606f2e6c6534b6900edb3f71c41a7b33afbaa/src/lib/index.js#L23-L79

**Comment by ifirmawan** [comment]
> Hi mas @ifirmawan, I saw we do the translation inside each of the field types. How if we do something like we implement on akvo react form, something like this one https://github.com/akvo/akvo-react-form/blob/d68606f2e6c6534b6900edb3f71c41a7b33afbaa/src/lib/index.js#L23-L79
> 
> I think we can handle the translation in one place, instead of doing that inside each field types.

Noted. thanks mas, Fixing in progress.

File: backend/json/603050002.json [hunk 1]
```diff
@@ -45,14 +84,26 @@
 					"required": true,
 					"meta": true,
 					"source": {
-						"file": "kenya_organizations.sqlite",
-						"parent_id": 0
-					}
+						"file": "cascade-296940912-v2.sqlite",
```
Comment (dedenbangkit) [inline backend/json/603050002.json]: wrong cascade @ifirmawan

File: backend/json/603050002.json [hunk 2]
```diff
@@ -3,22 +3,49 @@
 	"name": "Urban Sanitation",
 	"version": "1.0.0",
 	"cascades": [
-		"/sqlite/cascade-296940912-v2.sqlite",
-		"/sqlite/kenya_organizations.sqlite"
```
Comment (dedenbangkit) [inline backend/json/603050002.json:7]: why you remove this?

**Review by dedenbangkit — CHANGES_REQUESTED** [review]

File: backend/json/603050002.json (hunk 2 above)
Comment (ifirmawan) [reply backend/json/603050002.json:7]: sorry, I think I didn't merge the main before I did translations on this form

Comment (ifirmawan) [reply backend/json/603050002.json:7]: I've fixed it on this commit
https://github.com/akvo/nmis-mobile/pull/61/commits/0be5faa364ec933547a13c220bfb10cccf2cf4be

**Review by wayangalihpratama — APPROVED** [review]

**Comment by wayangalihpratama** [comment]
Hi, mas @dedenbangkit ,
I have reviewed this PR and pushed some updates for the TypeCascade mandatory icon and TypeOption dropdown value bug.

**Review by dedenbangkit — APPROVED** [review]

**Pull request merged**